scout/dashboard.md

```diff
@@ -79,6 +79,22 @@ Each entry in the list shows the following details:
 - The package type.
 - The number of images that use the package.
 
+### Images using base image
+
+<!-- TODO: Screenshot -->
+
+The **Images using base image** tab shows all images in an organization that use a specific base image.
+
+Each entry in the list shows the following details:
+
+- The repository name. Selecting the link opens [the list of tags for the repository](#repository-tag-list).
+- The registry location of the repository.
+- The most recent tag of the image and the vulnerabilities for that version. Selecting the link for the tag opens [the comparison view](#comparing-two-tag-images) for the repository.
```

Comment:
I don't think this is correct. There are no entry points directly to image comparison from this page https://www.figma.com/file/WJD5iilqvc9CjIbbCN3r6W/Docker-Scout-Site?type=design&node-id=832-160854&t=SC6dctwpYrofEvFO-4

Reply:
Clicking on the hyperlink in the 'Most recent image' column will bring users to the image layer detail view here https://www.figma.com/file/WJD5iilqvc9CjIbbCN3r6W/Docker-Scout-Site?type=design&node-id=832-147163&t=SC6dctwpYrofEvFO-4